pkg/utils/utils.go

```diff
@@ -344,7 +348,16 @@ func GetS3PETMFromParamsMap(params map[string]interface{}, logger logrus.FieldLo
 	// If the credentials are explicitly provided in params, use it.
 	// else let aws API pick the default credential provider.
 	var sess *session.Session
-	if _, ok := params[constants.AWS_ACCESS_KEY_ID]; ok {
+	var sessionOptions session.Options
+	caCert, ok := GetStringFromParamsMap(params, "caCert", logger)
+	if ok {
+		sessionOptions = session.Options{Config: aws.Config{
+			Region:      aws.String(region),
+		}}
+	}
+	var credential *credentials.Credentials
+	_, ok = params[constants.AWS_ACCESS_KEY_ID]
```

Comment:
Why do we have to split up the following line
```
if _, ok := params[constants.AWS_ACCESS_KEY_ID]; ok {
```
into two? Any benefit?

Reply:
Merge into one line to avoid propagating the ok value to the following context outside the if block.